Source/Neptune.Web/Views/TreatmentBMP/EditModelingAttributes.cshtml

```diff
@@ -56,12 +56,21 @@
                 <div class="row">
                     <div class="col-xs-12">
                         <div class="form-horizontal">
-                            @*<div class="form-group row">
+                        @*<div class="form-group row">
```

Comment:
Is there a reason we need to retain this commented out markup at 59-64?

Reply:
Not as far as I know. I'll  take it out